CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 0.3.0b0
 =======
+- Add development section on readme
```

Comment:
Would need to go under the unreleased section. In any case, considering that this is not a change in the code TBH I'm happy about just adding nothing here.